app-frontend/src/app/components/mosaicParameters/mosaicScenesParameters.controller.js

```diff
@@ -0,0 +1,143 @@
+import _ from 'lodash';
```

Comment:
Trying not to use lodash unless it's actually necessary - think there's only one place we use it right now and I'd like to keep it that way.